pkg/util/cnr.go

```diff
@@ -42,6 +42,8 @@ const (
 	CNRFieldNameTopologyZone           = "TopologyZone"
 	CNRFieldNameResources              = "Resources"
 	CNRFieldNameTopologyPolicy         = "TopologyPolicy"
+
+	ResourceRDMA = "vke.volcengine.com/rdma"
```

Comment:
Could you make the resource name configurable? We can assign the default value to `resource.katalyst.kubewharf.io`, and then pass in a custom value.